### ISSUE TITLE
Site Editor > Styles tab: Link directly with no style variations

### ISSUE THREAD

**Issue by jasmussen** [WordPress/gutenberg#52158]
The styles section is important to surface an entry point into editing global styles. For 6.3, however, the styles section will not come into [full fruition](https://github.com/WordPress/gutenberg/issues/50924#issuecomment-1580720872), and #50924 even details how the prominence of style variations is not necessarily ideal. 

So as to keep the value of surfacing the style editors, for 6.3 we'll keep the style variations. One bug we need to fix, however, is to ensure that if a theme doesn't have _any_ style variations, we skip the drilldown entirely and have the top level "Styles" section link directly to the global styles editor. This:

<img width="356" alt="Screenshot of the main site editor showing a Styles section" src="https://github.com/WordPress/gutenberg/assets/1204802/f43a03e2-de61-4a43-aa23-342f02485a44">

* If there are style variations, the "Styles" link here should navigate to the existing detail page showing style variations
* If there are _no_ style variations, the same link should go directly to the Edit View with the global styles panel open (same as clicking the edit button in the detail page)

**Comment by ntsekouras** [comment]
@jasmussen if I get the description right, is what we already have in trunk with [this PR](https://github.com/WordPress/gutenberg/pull/50573), right?

**Comment by jasmussen** [comment]
Oh you're right, this behavior is already in trunk. Here's what happens if you have style variations:

![style variations](https://github.com/WordPress/gutenberg/assets/1204802/f8d4b3a9-e51f-4f1e-a303-c70a6bc08ac3)

But when you have no style variations, the detail page is skipped:

![no style variations](https://github.com/WordPress/gutenberg/assets/1204802/030572ae-828c-4e23-b96e-af744f4a91cb)

In other words, you're right, this issue is already fixed.

Edit: I misunderstood initially.
